.github/workflows/ci.yml

```diff
@@ -49,6 +53,10 @@ jobs:
           if [ "${cc}" = "clang-10" ]; then
             sudo add-apt-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-10 main"
           fi
+          if [ "${cc}" = "clang-6.0" ]; then
+            # Problems compiling with Clang 6 against libstdc++-9
+            sudo apt-get remove libstdc++-9-dev libgcc-9-dev cpp-9
```

Comment:
Would it be enough to update system includes? I am okay with this solution anyway.

Reply:
I don't know how to do that tbh. clang finds all installations and always seems to select the latest version. I couldn't figure out how to force it to use another.